mfr/cli.py

```diff
@@ -19,7 +20,10 @@ def pip_install(path, filename):
     file_location = (os.path.join(path, filename))
 
     if os.path.isfile(file_location):
-        pip.main(['install', "-r", file_location])
+        pip_args = ['install', '-r', file_location]
+        if WHEELHOUSE_PATH:
```

Comment:
We should add a  `--wheelhouse` or `--no-wheelhouse` option to override this behavior. 

This doesn't have to get in before this is merged.